Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -877,3 +910,133 @@ - (void)vungleCreative:(nullable NSString *)creativeID readyForPlacement:(nullab
 }
 
 @end
+
+@implementation ALVungleMediationNativeAdAdapter
+
+- (instancetype)initVungleNativeAd:(id<MANativeAdAdapterDelegate>)nativeAdDelegate parameters:(id<MAAdapterResponseParameters>)parameters
+{
+    self = [super init];
+    if ( self )
+    {
+        self.nativeAdDelegate = nativeAdDelegate;
+        self.parameters = parameters;
+    }
+    return self;
+}
+
+- (void)requestNativeAd:(NSString *)placementIdentifier
+{
+    if ( ![[VungleSDK sharedSDK] isInitialized] )
+    {
+      return;
```

Comment:
Do you need to call a MAX delegate before this return?

Reply:
For failure to load

Reply:
also - alignment within the if block. The delegate method and return statement are not aligned.